core/frames.lua

```diff
@@ -165,6 +165,9 @@
    				TargetFrameToTTextureFrameTexture,
 				CastingBarFrame.Border,
 				TargetFrameSpellBar.Border,
+        MirrorTimer1Border,

```

Comment:
Seems like github shows different indentation to me. I have it aligned in my text editor(Atom)